libs/frontend/feature-render-builder/src/components/LinkTooltip.vue

```diff
@@ -56,36 +75,64 @@ const { link, componentId } = toRefs(props)
 const linkInput = ref()
 const editing = ref(false)
 const editedLink = ref('')
+const openInNewTab = ref(false)
 
 const linkDatalist = computed(() => getLinkDatalistOptions(site.value, componentId.value))
 
 const internalLink = computed(() => {
   return link.value?.startsWith('/') && link.value in site.value.pages
 })
 
+const component = computed(() => resolveComponent(site.value.context, componentId.value))
+
+const componentTargetInput = computed(
+  () => component.value?.inputs?.target?.is as string | undefined,
+)
+
 const gotoPage = () => {
   changePage(link.value)
 }
 
 const edit = async () => {
   editing.value = true
   editedLink.value = link.value
+  openInNewTab.value = componentTargetInput.value === '_blank'
   await nextTick()
   linkInput.value?.inputRef?.focus()
```

Comment:
Hi @sampullman 

The URL input in LinkTooltip gets auto-focused when edit button is clicked. This worked well before, but after anchors/pages datalist has been added to the input, the datalist is now covering the other UI (open in new tab checkbox and save button) when the list is too long. Even if we disable the auto-focus feature, the list will still cover the UI when user manually focus the input as long as there're enough items in the list.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/6deae992-04db-4760-8ad8-8c3c02b24202

Do you have an idea how can we improve this 🤔?

Reply:
I experimented a bit and couldn't get `datalist` styling to work as expected. I think we'll either need to use/adapt `PSMultiselect`, or maybe move the checkbox to the left side as a temporary improvement.

I'll merge this PR now though, because I don't think it's a critical problem. Selecting an item or pressing enter closes the datalist, which is ok.

Reply:
I tried to set the style of datalist with CSS yesterday, but it seems that the style of datalist is not customizable. I wasn't able to replace datalist with PSMultiselect in time, sorry about that! I'll include the change in another PR instead, thanks!

Reply:
Got it, no problem!